src/components/search/SearchBar/SearchBar.tsx

```diff
@@ -140,10 +140,17 @@ export default function SearchBar(props: Props) {
   }
 
   function handleKeyDown(event: React.KeyboardEvent<HTMLInputElement>) {
-    if (event.key === 'Enter' && inputValue === '') {
+    if (event.key === 'Enter') {
       event.preventDefault();
```

Comment:
Sorry for yet another question - if preventDefault and stopPropagation apply to all 'Enter' events, do they prevent selecting by pressing enter when there are autocomplete options? 

If so, could we change this so these only occur when loading or when inputValue is empty?